### PR TITLE
Paypal update: correct doc link, add note about supported devices

### PR DESCRIPTION
**Pull request by arxo**

### CODE REVIEW

File: entries/p/paypal.com.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
       "paypal-search.com"
     ],
     "mfa": "allowed",
-    "documentation": "https://www.paypal.com/us/smarthelp/article/faq4057",
+    "notes": "An eligible Apple or Android device is required to create a passkey.",
```
Comment (hkamran80) [inline entries/p/paypal.com.json:7]: I don't think this is needed, even though it's stated in the documentation. Do you know if passkey creation is actually limited to those devices or if it's more guidance than fact?

Comment (arxo) [reply entries/p/paypal.com.json:7]: I cannot check myself, but judging by Shannon Morse's guide video on the topic, it looks like it is indeed a restriction: https://www.youtube.com/watch?v=3jEm5WyN4EQ

Comment (Carlgo11) [reply entries/p/paypal.com.json:7]: I just checked, and sadly, it is true. There are no limitations that prevent them from accepting other browsers/platforms. In fact, I just set it up using Firefox and a user-agent switcher.

File: entries/p/paypal.com.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
       "paypal-search.com"
     ],
     "mfa": "allowed",
```
Comment (Carlgo11) [inline entries/p/paypal.com.json:6]: Passkeys seems to only be used by PayPal to replace the password.
```suggestion
    "passwordless": "allowed",
```

**Comment by arxo** [comment]
Can be reactivated.

**Comment by Carlgo11** [comment]
@arxo You haven't implemented the requested change.

**Comment by arxo** [comment]
Sorry, I missed that one. Done now.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**